letsencrypt/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.10.0
```

Comment:
The missing `4.9.0` is intentional, in case #1381 is merged before mine (which is more likely 🙂 )

Reply:
You'll need to update the version in the [config.json](https://github.com/home-assistant/hassio-addons/blob/master/letsencrypt/config.json#L3) as well, otherwise users won't be notified of a version update in their Home Assistant setup.

Not really sure if we should bump versions through PR's though. But since I couldn't find some kind of release management info I took the liberty of doing so ;)